src/logic/mutinyWalletSetup.ts

```diff
@@ -87,12 +87,17 @@ export async function setupMutinyWallet(settings?: MutinyWalletSettingStrings):
 
     const mutinyWallet = await new MutinyWallet("", undefined, proxy, network, esplora, rgs, lsp)
 
-    const nodes = await mutinyWallet.list_nodes();
+    let nodes = await mutinyWallet.list_nodes();
 
     // If we don't have any nodes yet, create one
     if (!nodes.length) {
-        await mutinyWallet?.new_node()
+        await mutinyWallet?.new_node();
+        nodes = await mutinyWallet?.list_nodes();
     }
 
+    const firstNode = nodes[0] as string || "";
+
+    await mutinyWallet?.start_nostr_wallet_connect(firstNode);
```

Comment:
Maybe instead of having it auto run on startup we can make a "start nwc" button in the same settings area as the model. 

My concern is that it's experimental and no way to stop someone from stealing all your funds. I would rather we keep playing with this.

Reply:
done